.gitignore

```diff
@@ -61,3 +61,8 @@ phpcs.xml
 # POT File
 languages/*
 !languages/README.md
+
+# @wordpress/scripts build directory
+build
+**/build.*
+**/build.asset.*
```

Comment:
Should anything be added to the `.distignore` file?

Reply:
I don't think so, I assume we want to keep the build folder for distribution since it contains the generated scripts.

Reply:
Yeah, the `/build` folder probably needs to be included in the distribution, instead of trying to mess with moving that target directory.

Reply:
@jonwaldstein For sure! I'm really thinking about if there are any specific files (file maps?) within the build that shouldn't be included in distribution.

Reply:
Gotchya, it appears wp scripts does not generate those file maps so we should be good.  If anything looks weird during distribution we can add to the distignore.